Tests/PackageLoadingTests/PD_4_2_LoadingTests.swift

```diff
@@ -887,6 +893,11 @@ class PackageDescription4_2LoadingTests: PackageDescriptionLoadingTests {
 
     // run this with TSAN/ASAN to detect concurrency issues
     func testConcurrencyNoWarmUp() throws {
+#if os(Windows)
+        // FIXME: does this actually trigger only on Windows or are other
+        // platforms just getting lucky?  I'm feeling lucky.
+        XCTSkip("Foundation Process.terminationStatus race condition (apple/swift-corelibs-foundation#4589")
```

Comment:
is this the right syntax? I think you need to throw the XCTSkip

Reply:
ugh, yes, missed the `throw`.